invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Item/ItemForms/ItemEditor/components/ItemForm/ItemForm.js

```diff
@@ -68,19 +62,23 @@ export class ItemForm extends Component {
   };
 
   render() {
+    const initialValues = this.props.data
+      ? this.prepareData(this.props.data.metadata)
+      : {};
     return (
       <BaseForm
-        initialValues={
-          this.formInitialData
-            ? this.prepareData(this.formInitialData.metadata)
-            : {}
-        }
+        initialValues={{
+          circulation_restriction: 'NO_RESTRICTION',
```

Comment:
In general, we should have a list of values in the config (same as backend) with a `default` value...
e.g.
```
circulation_restrictions: [
{ id: 'NO_RESTRICTION', label: 'No restrictions', default: true },
...
]
```